### ISSUE TITLE
NFRs - Customer Present - Process/method to prevent misuse/abuse

### ISSUE THREAD

**Issue by CDR-API-Stream** [ConsumerDataStandardsAustralia/standards-maintenance#11]
This request for clarification has been transferred from the main standards site.  Contents of the request, posted by @bmangalaganesh, are below:

Hi,

The specs define Customer Present as follows - Authenticated API requests made in direct response to interactions by the end customer using the digital services of the data recipient will be considered “Customer Present”.

The expected response times for customer present requests are significantly shorter than the unattended requests ( 1 or 1.5 seconds vs 4 seconds).

What is the mechanism/procedures that are kept in place to prevent misuse of customer present requests by ADRs?

In other words, the Data Holder when they receive any request with x-fapi-customer-ip-address header info present is expected to provider a faster response (while they do not truly know if there is a customer at the other end of the ADR/TPP app)

**Comment by CDR-API-Stream** [comment]
Enforcement mechanisms are beyond the scope of the standards.  They belong in the remit of the ACCC.

There is no technical mechanism that can be adopted by the standards to guarantee that participants (both Data Recipients and Data Holders) are providing true and accurate information under the standards.
